server/data/healthcheck.js

```diff
@@ -11,9 +11,8 @@ function dbCheck() {
 }
 
 const agentOptions = {
-  maxSockets: config.nomis.agent.maxSockets,
-  maxFreeSockets: config.nomis.agent.maxFreeSockets,
-  freeSocketTimeout: config.nomis.agent.freeSocketTimeout,
+  timeout: config.apis.hmppsAuth.agent.timeout,
```

Comment:
agent doesn't have `maxSockets` or `maxFreeSockets` options anyway.  It does have a `timeout` option though that is in the typescript template, so added that instead